packages/account/src/Components/terms-of-use/__tests__/terms-of-use.spec.tsx

```diff
@@ -166,8 +167,8 @@ describe('<TermsOfUse/>', () => {
         expect(mock_props.getCurrentStep).toHaveBeenCalledTimes(1);
         expect(mock_props.onCancel).toHaveBeenCalledTimes(1);
 
-        const agree_checkbox = screen.getByLabelText(agree_check);
-        const not_pep_checkbox = screen.getByLabelText(not_pep_check);
+        const agree_checkbox = screen.getByLabelText(agree_check) as HTMLInputElement;
```

Comment:
instead of type casting which is unsafe, we can have a type validation helper function. 
can you check this link please to see if we can apply it here? 
https://stackoverflow.com/questions/53195381/react-testing-library-with-typescript-set-an-inputs-value